### PR TITLE
Update gramatical mistake README.md

### PR DESCRIPTION
**Pull request by akshayakshay30**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
 <img src="./website/static/img/logo.svg" alt="Hermes logo - large H with wings" align="right" width="20%"/>
 
-Hermes is a JavaScript engine optimized for fast start up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
+Hermes is a JavaScript engine optimized for fast start-up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
 
 If you're only interested in using pre-built Hermes in a new or existing React Native app, you do not need to follow this guide or have direct access to the Hermes source. Instead, just follow [these instructions to enable Hermes](https://facebook.github.io/react-native/docs/hermes).
 
-If you want to know how to build and hack on Hermes directly, and/or integrate Hermes built from source into a React Native app then read on.
+If you want to know how to build and hack on Hermes directly, and/or integrate Hermes built from the source into a React Native app then read on.
```
Comment (avp) [inline README.md]: I don't think this change is necessary - "source" is frequently used without the definite article, particularly in this context.

Comment (rickhanlonii) [inline README.md:11]:
```suggestion
If you want to know how to build and hack on Hermes directly, and/or integrate Hermes built from source into a React Native app then read on.
```

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebook/hermes/blob/master/CONTRIBUTING.md)
 <img src="./website/static/img/logo.svg" alt="Hermes logo - large H with wings" align="right" width="20%"/>
 
-Hermes is a JavaScript engine optimized for fast start up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
+Hermes is a JavaScript engine optimized for fast start-up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
```
Comment (rickhanlonii) [inline README.md:7]:
```suggestion
Hermes is a JavaScript engine optimized for fast startup of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
```

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebook/hermes/blob/master/CONTRIBUTING.md)
 <img src="./website/static/img/logo.svg" alt="Hermes logo - large H with wings" align="right" width="20%"/>
 
-Hermes is a JavaScript engine optimized for fast start up of [React Native](https://reactnative.dev/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
+Hermes is a JavaScript engine optimized for fast startup of [React Native](https://reactnative.dev/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
```
Comment (rickhanlonii) [inline README.md:7]:
```suggestion
Hermes is a JavaScript engine optimized for fast start-up of [React Native](https://reactnative.dev/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
```

**Comment by ljharb** [comment]
What ever happened to this comment? https://github.com/facebook/hermes/pull/124#discussion_r332102268 
"startup" is more idiomatic english than "start-up".

**Comment by rickhanlonii** [comment]
I reverted my change when landing. What's idiomatic is subjective, but "start-up" being the [dictionary spelling](https://www.merriam-webster.com/dictionary/start-up) is objective so we went with start-up.

**Pull request closed without merging**